docs/src/user_docs/stdlib/mem.md

```diff
@@ -8,8 +8,9 @@ Module `std::mem` contains a set of utility procedures for working with random a
 
 | Procedure                              | Description   |
 | -------------------------------------- | ------------- |
+| `memcopy`                              | Copies `n` felts from `read_ptr` to `write_ptr`.<br/><br/>If possible, this procedure will copy word-aligned words internally using `memcopy_words` procedure, decreasing the total number of cycles required to make a copy.<br/><br/>It is advised to copy small chunks of memory using `memcopy_felts` procedure instead, since it won't have a computation logic overhead.<br/><br/>**Inputs:** `[n, read_ptr, write_ptr]`<br/>**Outputs:** `[]`<br/><br/>Total cycles:<ul><li>Read and write pointers are mutually unaligned: $23 + 14 * num\_felts$</li><li>Read and write pointers are mutually aligned: $66 + 14 * num\_prefix\_felts + 16 * num\_words + 14 * num\_suffix\_felts$</li></ul> |
```

Comment:
Q: Could we pinpoint the number of felts where we should prefer one to the other?